packages/itwinui-css/src/side-navigation/side-navigation.scss

```diff
@@ -102,7 +102,7 @@ $iui-side-navigation-icon-margins: calc(1.5 * var(--iui-size-m));
       }
     }
 
-    > span {
+    > span:last-of-type {
```

Comment:
To select the label span and not the new notification span

Reply:
I don't think we should be relying on dom position here.

Reply:
Earlier just the `span` selector worked since there was just one span under `.iui-sidenav-button`.

But now with the notification marker `span` also between `.iui-sidenav-button` and `.iui-button-icon` (as on [L132-145](https://github.com/iTwin/iTwinUI/blob/b23406cca9655588a3e474ee76a3c35d6b32f361/packages/itwinui-css/backstop/tests/side-navigation.html#L132-L145)), there are two spans under `.iui-sidenav-button` ([L140](https://github.com/iTwin/iTwinUI/blob/b23406cca9655588a3e474ee76a3c35d6b32f361/packages/itwinui-css/backstop/tests/side-navigation.html#L140) and [L149](https://github.com/iTwin/iTwinUI/blob/b23406cca9655588a3e474ee76a3c35d6b32f361/packages/itwinui-css/backstop/tests/side-navigation.html#L149)).

What alternative do you recommend to select the label span but not the notification span?

Reply:
Thanks, replaced with `> span:where(:not(.iui-notification-marker))`, since `.iui-notification-marker` is the span we want to exclude, and not `.iui-button-icon`

Reply:
Are you sure? I think `iui-button-icon` will be a parent of `iui-notification-marker`, so that's what we want to exclude.

Reply:
Ohh, right, but I think we need to exclude both, `.iui-button-icon` and `.iui-notification-marker`.

Because the parent for the icon is `.iui-button-icon` when there's no notification (E.g. [L181](https://github.com/iTwin/iTwinUI/blob/b23406cca9655588a3e474ee76a3c35d6b32f361/packages/itwinui-css/backstop/tests/side-navigation.html#L181)), but is `.iui-notification-marker` when there's a notification. (E.g. [L140](https://github.com/iTwin/iTwinUI/blob/b23406cca9655588a3e474ee76a3c35d6b32f361/packages/itwinui-css/backstop/tests/side-navigation.html#L140))

Changed it to `> span:where(:not(.iui-button-icon, .iui-notification-marker))`

Reply:
i don't think `iui-notification-marker` should ever be a parent of `iui-button-icon`. this is an impossible scenario in react because we wrap the icon prop (where the user will pass `NotificationMarker`) with `<span className='iui-button-icon'>`

Reply:
Ohh, I see, good point. I made the change like you suggested:

Old
![image](https://user-images.githubusercontent.com/45748283/200029688-13a5f2c5-632d-4a26-a14b-7b1875c87c08.png)

New
![image](https://user-images.githubusercontent.com/45748283/200029639-70740c28-c9f8-4193-a5b1-e93245ff045b.png)

Now `.iui-notification-marker` is never the parent of `.iui-button-icon`. Also updated the svg selector to what you suggested: `> span:where(:not(.iui-button-icon))`